validate-yaml.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const {load} = require('js-yaml');
+const Joi = require('joi');
+
+const permissionsSchema = Joi.object({
+    method: Joi.string().required(),
+    apps: Joi.array().items(Joi.string()),
+    args: Joi.array().items(Joi.any())
+})
+
+const frontendSchema = Joi.object({
+    title: Joi.string(),
+    reload: Joi.string(),
+    paths: Joi.array().items(Joi.string()),
+    sub_apps: Joi.array().items(Joi.object({
+        id: Joi.string().required().allow(''),
+        default: Joi.boolean(),
+        title: Joi.string(),
+        group: Joi.string(),
+        reload: Joi.string(),
+        permissions: permissionsSchema
```

Comment:
This can be either directly permissionsSchema or array
```suggestion
        permissions: [Joi.array().items(permissionsSchema), permissionsSchema]
```

Reply:
I assume the same applies at line 51. Permission for the whole app.

Reply:
Yup!